apps/site-landing/src/containers/App/styles.module.less

```diff
@@ -18,7 +18,7 @@
     grid-template-areas:
       "lefth header righth"
       "main  main   main"
-      "leftf footer rightf"
+      "lgap footer rgap"
```

Comment:
I've suddenly realized I setup the mobile landing without ensuring my base grid was correct (it's not)

Reply:
I don't care (for now), as it's fairly low on the list of priorities